src/integration-test/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MTurkRestApiTest.java

```diff
@@ -0,0 +1,73 @@
+package edu.kit.ipd.crowdcontrol.objectservice.crowdworking.mturk;
+
+import com.amazonaws.mturk.requester.doc._2014_08_15.Assignment;
+import com.amazonaws.mturk.requester.doc._2014_08_15.HIT;
+import com.amazonaws.mturk.requester.doc._2014_08_15.HITStatus;
+import edu.kit.ipd.crowdcontrol.objectservice.crowdworking.mturk.command.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.FileInputStream;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by marcel on 01.02.16.
+ */
+public class MTurkRestApiTest {
+    private MTurkConnection connection;
+    @Before
+    public void setUp() throws Exception {
+        Properties properties = new Properties();
+        properties.load(new FileInputStream("src/integration-test/resources/mturk.properties"));
+        connection = new MTurkConnection(
+                properties.getProperty("User"),
+                properties.getProperty("Password"),
+                "https://mechanicalturk.sandbox.amazonaws.com/");
+    }
+
+    @Test
+    public void testPublishHIT() throws Exception {
+        String id = new PublishHIT(connection,"Title1", "Description2",
+                0.20,60,2000,"test,for,everything",
+                2,2000000,"data").get();
+
+        assertNotEquals(id, null);
```

Comment:
First is always `expected`, second argument is `actual`.
http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(java.lang.Object, java.lang.Object)